app/views/foreman/smart_proxies/_content_tab.html.erb

```diff
@@ -0,0 +1,42 @@
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class=""><%= s_("Content View Name") %></th>
+      <th class=""><%= s_("Composite") %></th>
+      <th class=""><%= s_("Last Published") %></th>
+      <th class=""><%= s_("Hosts") %></th>
+      <th class=""><%= s_("Products") %></th>
+      <th class=""><%= s_("Yum repos") %></th>
+      <th class=""><%= s_("Docker repos") %></th>
+      <th class=""><%= s_("Packages") %></th>
+      <th class=""><%= s_("Errata") %></th>
+      <th class=""><%= s_("Puppet modules") %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr ng-repeat-start="env in syncStatus.lifecycle_environments">
+      <td colspan="10"
+        ng-click="toggleExpandEnvironment(env)"
+        class="expander {{isEnvronmentExpanded(env) ? '' : 'collapsed'}}">
+        <span class="caret"></span>
+        {{ env.name }}
+      </td>
+    </tr>
+    <tr ng-repeat-end ng-repeat="cv in env.content_views" ng-show="isEnvronmentExpanded(env)">
+      <td>
+        <a href="/content_views/{{cv.id}}/versions" target="_self">
```

Comment:
This could be written using states instead (https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/content-views.routes.js#L70):

```
ui-sref="content-views.versions({versionId: cv.id)"
```